### PR TITLE
LPS-153304 Limit characters in Save Fragment Composition's inputs and add description field validation

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java [hunk 1]
```diff
@@ -75,6 +76,7 @@ public FragmentComposition addFragmentComposition(
 			fragmentCompositionKey);
 
 		validate(name);
+		validateDescription(description);
 		validateFragmentCompositionKey(groupId, fragmentCompositionKey);
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java:80]: Just noticed, that although it was already there validateFragmentCompositionKey is not sorted according to service.xml.
Can you sort this one too in a separate commit?

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Comment by ozysouza** [comment]
ci:forward

**Comment by DiegoHu97** [comment]
Resent here: https://github.com/liferay-echo/liferay-portal/pull/8440

**Pull request closed without merging**